src/Symfony/Component/Utf8/CodePoints.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Utf8;
+
+use Symfony\Component\Utf8\Exception\InvalidArgumentException;
+
+final class CodePoints implements GenericStringInterface, \Countable
```

Comment:
Well we did it at the beginning and we finally removed them to keep the API light and focused on the most important functions. This can be discussed anyway :)